Lib/test/test_asyncio/test_tasks.py

```diff
@@ -8,6 +8,7 @@
 import random
 import re
 import sys
+from threading import Event
```

Comment:
I'd move this import into the test function -- otherwise it could be confused with asyncio.Event.